js/main.js

```diff
@@ -0,0 +1,37 @@
+var $form = document.querySelector('form');
+var $avatarImg = document.querySelector('img');
+var $avatarUrlInput = document.querySelector('#avatarUrl');
+var $usernameInput = document.querySelector('#username');
+var $fullNameInput = document.querySelector('#fullName');
+var $locationInput = document.querySelector('#location');
+var $bioTextarea = document.querySelector('#bio');
+
+function updateAvatar(event) {
+  if ($avatarUrlInput.value === '') {
+    $avatarImg.setAttribute('src', 'images/placeholder-image-square.jpg');
+  } else {
+    $avatarImg.setAttribute('src', event.target.value);
+  }
+}
+
+$avatarUrlInput.addEventListener('input', updateAvatar);
+
+function submit(event) {
+  event.preventDefault();
+  $avatarImg.setAttribute('src', 'images/placeholder-image-square.jpg');
+  data.profile.avatarUrl = $avatarUrlInput.value;
+  $avatarUrlInput.value = '';
```

Comment:
Resetting your form can be done more succinctly:

https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset